setup.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from setuptools import setup
+
+__version__ = "1.0.2"
```

Comment:
I don't think that is a default value for all plugins (in fact, I think it is only used in the deblur plugin because it is matching the deblur version that it implements). Probably using the versioning from Q2 would be better.